changelog.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v0.4.0 | 2019-12-05
+
+* Add calibration data as attribute of force channels.
+* Take into account slicing of channels when returning force calibration data.
+* Fixed slicing behaviour when slicing continuous time channels within a timestep.
+* Export pixel size and dwell time to TIFFs exported with pylake.
+* Implement slicing for kymographs.
```

Comment:
A short tutorial entry would be nice. And to point to that from the changelog. When someone comes from a previous version and looks at the changelog, if they see an interesting feature, they would like to use it. And it can be disorienting if there is no hint of how to use the new feature.

Reply:
Done. I made a separate page for kymos. Despite that this duplicates some stuff from scans, I think it is nice to have a head called kymographs, since their implementation is bound to diverge.